### ISSUE TITLE
Steps to integrate in Superset

### ISSUE THREAD

**Issue by mksbiz** [cancan101/graphql-db-api#136]
Great work, this is very helpful to refer for integrating non-supported data sources in Superset

Can you please help with how to integrate this with Superset. In Superset repo they have : https://github.com/apache/superset/tree/master/superset/db_engine_specs
which also includes the file : https://github.com/apache/superset/blob/master/superset/db_engine_specs/shillelagh.py

Can you provide steps of how you integrated your codebase into Superset. What files from your codebase were used, imported, and location where they were included in Superset codebase. Thanks!

**Comment by cancan101** [comment]
In a [subsequent blog post](https://blog.alexrothberg.com/2022/05/18/creating-sqlalchemy-dialect-for-airtable/), I do address this. See the section: "Database Engine Specs (Superset)". This package / repos does also have a db_engine_spec: https://github.com/cancan101/graphql-db-api/blob/main/graphqldb/db_engine_specs.py

TL; DR, you should be able to just install the package and have Superset recognize it.

**Comment by MathyV** [comment]
Hi,

I wanted to test Superset with this package, just as a fyi for whomever comes across this issue, these are the steps I took to get it integrated using the docker-compose setup:

```
git clone https://github.com/apache/superset.git
cd superset
git checkout 2.0.0
touch docker/requirements-local.txt
echo "google" > docker/requirements-local.txt
echo "sqlalchemy-graphqlapi" > docker/requirements-local.txt
TAG=2.0.0 docker-compose -f docker-compose-non-dev.yml pull
TAG=2.0.0 docker-compose -f docker-compose-non-dev.yml up
```

**Comment by cancan101** [comment]
Thanks for posting. I am curious as to why you needed the `google` dependancy. What happened without that?

**Comment by MathyV** [comment]
> Thanks for posting. I am curious as to why you needed the `google` dependancy. What happened without that?

Sorry, I missed this question. Without the google package installed it throws exceptions for missing imports whenever I tried to add the graphql database. Perhaps you have it installed already through another package?

**Comment by cancan101** [comment]
I would be curious to see the error that is thrown if `google` is not installed. I am not aware of any dependency that requires the package. Closest I know if is `shillelagh` needing `google-auth`: https://github.com/betodealmeida/shillelagh/blob/8edf3ed24b315baa86e72d19aee5511a0b6f1b87/setup.cfg#L101
